backend/webapi/Features/AccessRequests/ImmsBC.cs

```diff
@@ -0,0 +1,108 @@
+namespace Pidp.Features.AccessRequests;
+
+using DomainResults.Common;
+using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+using NodaTime;
+
+using Pidp.Data;
+using Pidp.Infrastructure.HttpClients.Keycloak;
+using Pidp.Infrastructure.HttpClients.Mail;
+using Pidp.Infrastructure.HttpClients.Plr;
+using Pidp.Infrastructure.Services;
+using Pidp.Models;
+using Pidp.Models.Lookups;
+
+public class ImmsBC
+{
+    public static IdentifierType[] AllowedIdentifierTypes => [IdentifierType.PhysiciansAndSurgeons, IdentifierType.Pharmacist, IdentifierType.Nurse];
+    public class Command : ICommand<IDomainResult>
+    {
+        public int PartyId { get; set; }
+    }
+
```

Comment:
Not a huge deal, but could you please add a `CommandValidator` here like other access requests?